components/TextFields/src/MDCTextField.m

```diff
@@ -792,20 +792,39 @@ - (void)mdc_setAdjustsFontForContentSizeCategory:(BOOL)adjusts {
   [_fundament mdc_setAdjustsFontForContentSizeCategory:adjusts];
 }
 
-- (NSString *)accessibilityValue {
+/*
+ Returns a combination of the following:
+ -  The superclass `accessibilityLabel` value
```

Comment:
_Question:_ What was the motivation behind using \`accessibilityLabel\` instead of @c accessibilityLabel?

Reply:
In non-doc comments? I think it's just easier to read since we're mostly familiar with Markdown.